include/swift/SIL/SILBuilder.h

```diff
@@ -2125,6 +2125,15 @@ class SILBuilder {
 #endif
   }
 
+  bool isLoadableOrOpaque(SILType Ty) {
+    if (!F) {
+      // We are inserting into the static initializer of a SILGlobalVariable.
+      // All types used there are loadable by definition.
```

Comment:
What if the emitter for the initializer has a bug? I think you still want to use ResilienceExpansion::Maximal here

Reply:
Unfortunately this does not work. The conservative assumption would trigger a false assert.

Reply:
Oh wait, you are saying ResilienceExpansion::Maximal. Yeah, that's a good idea.